frontend/src/scenes/pipeline/PipelineAppConfiguration.tsx

```diff
@@ -0,0 +1,122 @@
+import { LemonSkeleton, LemonWidget, Tooltip } from '@posthog/lemon-ui'
+import { useActions, useValues } from 'kea'
+import { Form } from 'kea-forms'
+import { Field } from 'lib/forms/Field'
+import { IconLock } from 'lib/lemon-ui/icons'
+import { LemonMarkdown } from 'lib/lemon-ui/LemonMarkdown'
+import React, { useEffect, useState } from 'react'
+import {
+    defaultConfigForPlugin,
+    determineInvisibleFields,
+    determineRequiredFields,
+    getConfigSchemaArray,
+    isValidField,
+} from 'scenes/pipeline/configUtils'
+import { PluginField } from 'scenes/plugins/edit/PluginField'
+
+import { pipelineAppLogic } from './pipelineAppLogic'
+
+export function PipelineAppConfiguration(): JSX.Element {
+    const { appType } = useValues(pipelineAppLogic)
+
+    if (appType === 'webhook') {
```

Comment:
do it by if appType === 'batch_export' as webhooks and transformation apps work the same way